docs/source/geometry.rst

```diff
@@ -16,13 +16,13 @@ kornia.geometry
 Coordinates Subpixel
 --------------------
```

Comment:
should we move this under `kornia.geomtetry.coords` ?

Reply:
I don't have a strong opinion either way. I don't think that it should be done as part of this PR though.

Reply:
lets do it in a separated PR